Resources/ServerInfo/Guidebook/ServerRules/RulesSL/CoreRules/RuleSL2.xml

```diff
@@ -1,23 +1,17 @@
 <Document>
- # Rule 2 – No griefing or harassment.
- Intentionally ruining the gameplay of other players is forbidden. Round ending actions, such as:
+ # Rule 2 – No Griefing, Bug Abuse, or Exploits
+ Intentionally ruining gameplay for others is prohibited. Round-ending or highly disruptive actions, such as:
 
-  - Releasing the tesla
+- Releasing the Tesla/Singularity
+- Large-scale station destruction (department-size or larger)
+- RDM (Random Death Match)
+- Intentionally Plasma or Tritium flooding the station
+- May result in a permanent ban. Some objectives or circumstances may allow exceptions—when in doubt, ask an admin before proceeding.
 
-  - Significant station damage (department size+)
-  - Mass round removal, plasma flooding
-  - Mass RDM (Random Death Match)
+Additionally, using bugs/exploits to gain an unfair advantage is prohibited. Please report them via ahelp or in the Discord.
```

Comment:
maybe include some examples here, like using camera rotating to gain a unfair FOV

Reply:
eh camera rotation is good, maybe some form of anti spam feature though in the future isn't a bad idea